maison-de-l-orient-et-de-la-mediterannee.csl

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded">
+<info>
+    <title>Maison de l'Orient et de la Méditerranée (French)</title>
+    <id>http://www.zotero.org/styles/maison-de-l-orient-et-de-la-mediterannee</id>
+    <link href="http://www.zotero.org/styles/maison-de-l-orient-et-de-la-mediterannee" rel="self"/>
+    <link href="http://www.zotero.org/styles/collection-de-l-ecole-francaise-de-rome-full-note" rel="template"/>
+    <link href="http://www.mom.fr/sites/mom.fr/files/img/Publications/recommandationsauteurs.pdf" rel="documentation"/>
+    <author>
+      <name>Magali Lugnot</name>
+      <email>magali.lugnot@mom.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <issn>1274-6525</issn>
+    <eissn>1955-4982</eissn>
```

Comment:
@Lugnot, only publications/journals can have ISSNs. Since this style is named after the organization, `<issn>1274-6525</issn>` and `<eissn>1955-4982</eissn>` should be deleted.

(once this style is accepted, we could always make a "dependent" CSL style for the journal "Travaux de la Maison de l'Orient et de la Méditerranée" that these ISSNs belong to, but we can discuss that later)